lib/models/repository-states/present.js

```diff
@@ -119,6 +119,8 @@ export default class Present extends State {
       const includes = (...segments) => fullPath.includes(path.join(...segments));
 
       if (filePathEndsWith(fullPath, '.git', 'index')) {
+        keys.add(Keys.stagedChanges);
+        // todo: stagedChangesSinceParentCommit appears to be dead code and can be removed
```

Comment:
🚨 Unresolved TODO

Reply:
this is some unrelated dead code we discovered, but due to the possibility of introducing additional bugs I say we punt on this unrelated change and clean it up later.

Reply:
Based on my knowledge of how this data was used, and looking at where/how it's still being referenced in the codebase, I'm confident that it's safe to clean it up without fear of introducing bugs. I'll go ahead and do so.